src/Compilers/CSharp/Portable/Symbols/SymbolWithAnnotations.cs

```diff
@@ -37,7 +37,7 @@ internal abstract class TypeSymbolWithAnnotations : IFormattable
         // PROTOTYPE(NullableReferenceTypes): consider removing this method and using Create below (which handles nullable value types).
         internal static TypeSymbolWithAnnotations CreateUnannotated(INonNullTypesContext nonNullTypesContext, TypeSymbol typeSymbol, ImmutableArray<CustomModifier> customModifiers = default)
         {
-            return Create(typeSymbol, nonNullTypesContext, isAnnotated: false, customModifiers.NullToEmpty());
+            return Create(typeSymbol, nonNullTypesContext, isAnnotated: false, treatUnconstrainedGenericsAsNullable: false, customModifiers.NullToEmpty());
```

Comment:
>treatUnconstrainedGenericsAsNullable [](start = 79, length = 36)

Perhaps `treatUnconstrainedTypeParameterAsNullable`, since this applies to explicit type parameter instances, not nested type parameters.